pages/[prefix]/explore/collectibles.vue

```diff
@@ -15,6 +16,22 @@ const checkRouteAvailability = () => {
   }
 }
 
+const getSeoMeta = computed(() => {
+  const prefix = urlPrefix.value
+  const isAssetHub =
+    Object.keys(chainNameSeoMap).includes(prefix) && assetHub.includes(prefix)
+
+  return {
+    title: isAssetHub
+      ? `Collections on Asset Hub ${getSeoPrefixName(prefix)}`
```

Comment:
seems this one doesn't work?

![image](https://github.com/kodadot/nft-gallery/assets/5887929/8302b1b5-04bf-4ec5-9ed3-ef1d32585dac)

Reply:
yes, yet not sure how crawler bots will see it, let's give it shot